ivy/functional/backends/tensorflow/experimental/statistical.py

```diff
@@ -66,6 +66,26 @@ def nanmean(
     return tf.experimental.numpy.nanmean(a, axis=axis, keepdims=keepdims, dtype=dtype)
 
 
+def nanmin(
+    a: Union[tf.Tensor, tf.Variable],
+    /,
+    *,
+    axis: Optional[Union[int, Tuple[int]]] = None,
+    keepdims: Optional[bool] = False,
+    out: Optional[Union[tf.Tensor, tf.Variable]] = None,
```

Comment:
different backend functions have been given different order of arguments (e.g. `out` is at the end in the `paddle` backend implementation but in the middle in the `tensorflow` backend implementation). Although this doesn't affect usage because all are keyword arguments, we should stick to a consistent function signature across the backends

Reply:
Okay noted. Done in the latest commit